ee/codegen/src/generators/protected-python-file.ts

```diff
@@ -0,0 +1,309 @@
+// This file is a direct copy paste of
+// https://github.com/fern-api/fern/blob/main/generators/python-v2/ast/src/PythonFile.ts
+// But the only difference is that all private methods are now protected
```

Comment:
Would skip reviewing this file

Reply:
If we go forward with this, we should update the upstream fern package, but this allows us to be unblocked for the time being